src/lib/es5.d.ts

```diff
@@ -2051,2271 +2049,110 @@ interface Int8ArrayConstructor {
      * Returns a new array from a set of elements.
      * @param items A set of elements to include in the new array object.
      */
-    of(...items: number[]): Int8Array;
+    of(...items: T[]): A;
 
     /**
      * Creates an array from an array-like or iterable object.
      * @param arrayLike An array-like or iterable object to convert to an array.
      */
-    from(arrayLike: ArrayLike<number>): Int8Array;
+    from(arrayLike: ArrayLike<T>): A;
 
     /**
      * Creates an array from an array-like or iterable object.
      * @param arrayLike An array-like or iterable object to convert to an array.
      * @param mapfn A mapping function to call on every element of the array.
      * @param thisArg Value of 'this' used to invoke the mapfn.
      */
-    from<T>(arrayLike: ArrayLike<T>, mapfn: (v: T, k: number) => number, thisArg?: any): Int8Array;
+    from<U>(arrayLike: ArrayLike<U>, mapfn: (v: U, k: number) => bigint, thisArg?: any): BigInt64Array;

```

Comment:
should that not be something like:
```suggestion
    from<U>(arrayLike: ArrayLike<U>, mapfn: (v: U, k: number) => T, thisArg?: any): A;
```